insights/parsers/pmlog_summary.py

```diff
@@ -70,24 +75,43 @@ class PmLogSummary(CommandParser, dict):
         kernel.all.cpu.nice  0.000 none
         kernel.all.cpu.steal  0.000 none
         kernel.all.cpu.idle  3.986 none
+        kernel.all.pressure.io.full.avg ["10 second"] 0.001 none
```

Comment:
@xiangce based on these [metrics](https://gitlab.cee.redhat.com/pgarciaq/rosrules/-/blob/master/samples/aws2json.py#L90-L96), do we need to keep the older ones as is?

Reply:
@apuntamb I think it's OK, since it's just an example here.